lib/src/generator/generator_utils.dart

```diff
@@ -34,6 +34,16 @@ String generateCategoryJson(Iterable<Categorization> categories, bool pretty) {
   return encoder.convert(indexItems.toList());
 }
 
+String regEx(String? input){
```

Comment:
Hmm I'm not sure about the names of different things in here. Can you look into naming the function differently, and the variables inside, and maybe add a doc comment above?

https://dart.dev/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types

Reply:
Addressed in new commit